gallery/plot_repurposing_annotations.py

```diff
@@ -1,75 +1,204 @@
 """
-=======================
-Repurposing annotations
-=======================
-
-The following example illustrates the operations available in the torchvision.ops module for repurposing object
-localization annotations for different tasks (e.g. transforming masks used by instance and panoptic segmentation
+=====================================
+Repurposing masks into bounding boxes
+=====================================
+
+The following example illustrates the operations available in the
+`torchvision.ops <https://pytorch.org/vision/stable/ops.html>`_ module for repurposing
```

Comment:
instead of using hard links we should instead use the sphinx directives which will preserve the current doc version that's being browsed. In this case this should be

```suggestion
:mod:`torchvision.ops` module for repurposing
```

Reply:
:mod:`torchvision.ops` seems to not render a HTML link.

![image](https://user-images.githubusercontent.com/47158509/135129174-cec6fa69-81d0-4c81-85d8-6e11470ccb11.png)

while the previous one rendered a link, which can be redirected.

![image](https://user-images.githubusercontent.com/47158509/135129555-4531523f-5427-4221-9dce-ed4df3eb6066.png)

I would prefer a link, since that can be clicked.

Reply:
Isn't it actually a link? It should be. If it's not, maybe try ``` :mod:`ops` ``` then. 
Otherwise let's do what we did here: https://github.com/pytorch/vision/blob/main/gallery/plot_transforms.py#L6-L7
 and https://raw.githubusercontent.com/pytorch/vision/main/docs/source/transforms.rst (see the `.. _transforms:` at the top )

Reply:
Done, finally the transforms trick, worked. :ref: has worked and we now get redirect links.